VideoApi/VideoApi.Contract/Responses/ParticipantInHearingResponse.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace VideoApi.Contract.Responses
 {
-    public class JudgeInHearingResponse
```

Comment:
Would this be better suited as HostInHearingResponse?

Reply:
That was my initial inclination, but I went against it as if you look at the properties in this class, there is nothing restricting it to just hosts, the same way there was nothing restricting it to just judges in the first place. By keeping it generic which is what the class supports, it will save having to refactor it in the future (again) if needed, as well as accurately describing what the class accommodates. 

It was a bit of a red herring in the beginning when it was named as a Judge when the class itself doesn't restrict itself to judges.